doc/sphinx/make-your-own-compositor.md

```diff
@@ -0,0 +1,248 @@
+---
+discourse: 4911,5164,5603,6756,8037
+---
+
+# Using Mir Libraries to Build Your Own Wayland Compositor
+If you've seen [the demo tutorial](explore-mir-using-demos), you're probably
+taken away by Mir's capabilities and want to use it and the supporting
+libraries to build your own Wayland compositor. This section will guide you
+through the installation and basic usage of Mir. You'll be even more impressed
+by how easy it is to create a Wayland compositor using Mir!
+
+### A barebones Mir compositor
+
+The following code block is the bare minimum you need to run a Mir compositor:
```

Comment:
I think that this is the wrong order of things. A tutorial should typically be linear in how its followed. I would recommend the following order:

1. Installation
2. Programming
3. Building
4. Running

Reply:
Agreed